apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Getting started
+description: Getting started with Hopper UI.
+---
+
+{/* <Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout> */}
```

Comment:
is this commented out? how come?

Reply:
Good call, I thought it was not necessary anymore with the way I restructured things